src/node_contextify.cc

```diff
@@ -387,11 +387,15 @@ class ContextifyContext {
     Local<Object> proxy_global = PersistentToLocal(isolate,
                                                    ctx->proxy_global_);
 
-    bool in_sandbox = sandbox->GetRealNamedProperty(property).IsEmpty();
+    bool in_sandbox = !(sandbox->GetRealNamedProperty(property).IsEmpty());
     bool in_proxy_global =
-        proxy_global->GetRealNamedProperty(property).IsEmpty();
-    if (!in_sandbox || !in_proxy_global) {
-      args.GetReturnValue().Set(None);
+      !(proxy_global->GetRealNamedProperty(property).IsEmpty());
+    if (in_sandbox || in_proxy_global) {
+      v8::PropertyAttribute attr = None;
+      if (in_sandbox) {
+        attr = sandbox->GetPropertyAttributes(property);
+      }
```

Comment:
Got it. Also given that per #855 I want to at least attempt to rip out proxy global entirely that might end up being unnecessary.

Probably want to leave a TODO(smikes) in the source code though for later.